targets/esp8266/user/jerry_extapi.c

```diff
@@ -92,25 +84,29 @@ DELCARE_HANDLER(print) {
     printf ("\r\n");
   }
   return jerry_create_boolean (true);
-}
-
-
-/*---------------------------------------------------------------------------*/
+} /* print */
 
 DELCARE_HANDLER(gpio_dir) {
   int port, value;
```

Comment:
Just about everywhere in this source: please move all variable declarations closest to their first use, preferably merging declaration and initialisation.